### PR TITLE
Improved documentation for predefined transitions

### PR DESCRIPTION
**Pull request by IgorKvasn**
Hi,
It is really sad to see a great library that deals with animations and it does not provide an interactive showcase for **all** of these animations. Right now,iIndividual transitions are just described in a very brief textual way, but that is I guess not enough :(  People want to actually see these transitions in action before they decide to use them in their own project. I know... some of them are used here and there on the documentation page, but not all of them and (more importantly) not in one place.
I also found 2 transitions that weren't mentioned anywhere (_scrollThen_ and _scale_), so I created a documentation for them, too.

btw. sorry for the way I implemented tabs in _predefined.hbs_ template... I guess it would be much nicer to create nested routes for each of them and not cram averything into one template... but I am just too lazy...

### CODE REVIEW

File: tests/dummy/app/controllers/transitions/predefined.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Ember from "ember";
+
+export default Ember.Controller.extend({
+  showDetail: false,
+  activeTab: 'toLeft',
+
+  actions: {
+    toggleDetail: function() {
```
Comment (Kuzirashi) [inline tests/dummy/app/controllers/transitions/predefined.js]: Possible use of ES2015: `toggleDetail() {`

File: tests/dummy/app/transitions.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import Ember from "ember";
 
-export default function(){
+export default function () {
```
Comment (Kuzirashi) [inline tests/dummy/app/transitions.js]: IMO the standard is: `function() {` so only 1 space should be added.

File: tests/dummy/app/transitions.js [hunk 3]
```diff
@@ -80,12 +80,53 @@ export default function(){
...
+
+  // BEGIN-SNIPPET scale-demo
+  this.transition(
+    this.hasClass('scale-demo'),
+    this.use('scale', {duration})
+  );
+  // END-SNIPPET
+
+
+/*  this.transition(
```
Comment (Kuzirashi) [inline tests/dummy/app/transitions.js]: Shouldn't this be deleted?

File: tests/dummy/app/transitions.js (hunk 2 above)
Comment (IgorKvasn) [inline tests/dummy/app/transitions.js]: I guess my IDE formatted it this way, I'll remove the space after "function"

File: tests/dummy/app/transitions.js (hunk 3 above)
Comment (IgorKvasn) [inline tests/dummy/app/transitions.js]: yes, you're right

**Comment by ef4** [comment]
Thanks, this is great.

**Pull request merged**